api/handler/v1beta1/adapter.go

```diff
@@ -393,6 +407,24 @@ func (a *adapter) ToPolicyProto(p *domain.Policy) (*guardianv1beta1.Policy, erro
 	return policyProto, nil
 }
 
+func (a *adapter) ToPolicyAppealConfigProto(p *domain.Policy) *guardianv1beta1.PolicyAppealConfig {
```

Comment:
```suggestion
func (a *adapter) ToPolicyAppealConfigProto(p *domain.Policy) *guardianv1beta1.PolicyAppealConfig {
	if p.Appeal == nil {
		return nil
	}
```

Reply:
This will not work as `p.Appeal` is not a slice, it is a struct.

Reply:
now that `p.AppealConfig` is a pointer, added this check.